### DISCUSSION TITLE
emelator is stop working

### DISCUSSION THREAD

**Discussion by emadalslahi** [Genymobile/feedback#21] [Genymotion Desktop] [answered]
![Screenshot 2022-11-05 221426](https://user-images.githubusercontent.com/41680195/200137081-ca9bcbd2-5cc1-49ae-826c-69703f224cb5.png)
i started word yesterday with 3 devices in emulator and they run good
today suddenly they stoped and keep showing errors
<html><body><p>Failed to open a session for the virtual machine <b>Samsung Galaxy S10</b>.</p><qt><p>Failed to open/create the internal network <b><nobr>'HostInterfaceNetworking-VirtualBox Host-Only Ethernet Adapter #2'</nobr></b> (VERR_INTNET_FLT_IF_NOT_FOUND).</p><p>Failed to attach the network LUN (VERR_INTNET_FLT_IF_NOT_FOUND).</p>

Result Code: | E_FAIL (0x80004005)
-- | --
Component: | ConsoleWrap
Interface: | IConsole {872da645-4a9b-1727-bee2-5585105b9eed}

<br></body></html>

really this application is worth no thing its very bad and i have west time and effort on it..

**Comment by jmaneyrol69** [comment, marked as answer]
Hi @emadalslahi,

We are sorry but this discussion board is not suitable for technical support requests.
Please contact our tech support team at https://support.genymotion.com/hc/en-us/requests/new

Thank you for your understanding.
Best regards
